mail/templates/comments/body.html

```diff
@@ -1,48 +1,34 @@
 {% extends "email_base.html" %}
-{% load humanize %}
-{% load timeago %}
 
-{% block title %}Open Discussions - New Comment{% endblock %}
-{% block styles %}
-<style type="text/css">
-</style>
-{% endblock %}
+{% block title %}Re: [{{ channel.name|safe }}]  post.title|safe %}{% endblock %}
 
 {% block content %}
-
-<table width="100%" cellpadding="13" cellspacing="0" border="0">
-  <tbody>
-    <tr>
+<!-- 1 Column Text : BEGIN -->
+  <tr>
       <td>
-        <div>
-          <p>From {{ comment.author_name }}:</p>
-          <p class="comment-content">
-            {{ comment.text|truncatewords:100 }}
-            {% if is_comment_reply %}
-              <a href="{{ base_url }}{% url 'channel-post-comment' channel_name=post.channel_name post_id=post.id post_slug=post.slug comment_id=comment.id %}">(read more)</a>
-            {% else %}
-              <a href="{{ base_url }}{% url 'channel-post' channel_name=post.channel_name post_id=post.id post_slug=post.slug %}">(read more)</a>
-            {% endif %}
-          </p>
-        </div>
+          <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+              <tr>
+                  <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
+                      <p style="margin: 0 0 10px 0;">{{ comment.text }}</p>
+                  </td>
+              </tr>
+          </table>
       </td>
-    </tr>
-  </tbody>
-</table>
-<div class="read-more alignleft" style="margin-left: 13px; margin-top: 13px;">
-  {% if is_comment_reply %}
-  <a href="{{ base_url }}{% url 'channel-post-comment' channel_name=post.channel_name post_id=post.id post_slug=post.slug comment_id=comment.id %}" class="btn-primary">View on Open Discussions</a>
-  {% else %}
-  <a href="{{ base_url }}{% url 'channel-post' channel_name=post.channel_name post_id=post.id post_slug=post.slug %}" class="btn-primary">View on Open Discussions</a>
-  {% endif %}
-</div>
+  </tr>
+<!-- 1 Column Text : END -->
 {% endblock %}
 
 {% block footer %}
-<p>
-  You are receiving this email because you are following a post on Open Discussions.
-  <br/>
-  If you don't want to receive these emails in the future, you can
-  <a href="{{ base_url }}{% url 'channel-post' channel_name=post.channel_name post_id=post.id post_slug=post.slug %}">unfollow this post</a>.
-</p>
+You are receiving this email because you are following a post on Open Discussions.
```

Comment:
`Open Discussions` needs to be the configurable site name.

Reply:
Can we make this `{{ site_name }}` instead of `Open Discussions`?

Reply:
Yup, sorry, we have so many permutations of this now ("MIT Open", "Open Discussions", maybe more?), can be difficult to catch them all.